bin/reth/src/commands/debug_cmd/merkle.rs

```diff
@@ -278,7 +278,7 @@ impl Command {
                     let clean_result = merkle_stage.execute(&provider_rw, clean_input);
                     assert!(clean_result.is_ok(), "Clean state root calculation failed");
                     if clean_result.unwrap().done {
-                        break;
+                        break
```

Comment:
please add back, unrelated changes

Reply:
my formatter mistakenly did this, have corrected